### PR TITLE
CSS et al: Rebranding

### PR DESCRIPTION
**Pull request by matevz**
This PR
- changes the docs logo and favicon to a new themable svg
- adapts the CSS for the new Oasis branding
- adds the new Roboto fonts
- updates the token rewards static svg (followup to https://github.com/oasisprotocol/docs/pull/774)
- bumps docusaurus to 3.2.1

[Preview>>](https://deploy-preview-790--oasisprotocol-docs.netlify.app/)

### CODE REVIEW

**Review by kaja-osojnik — APPROVED** [review]

File: static/img/logo.svg [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      .cls-1 {
+        fill: #0500e2;
+        stroke-width: 0px;
+      }
+      @media (prefers-color-scheme: dark) {
+        .cls-1 {
+          fill: white;
+          stroke-width: 0px;
+        }
+      }
```
Comment (lukaw3d) [inline static/img/logo.svg]: Interesting magic

File: src/css/custom.css [hunk 2]
```diff
@@ -6,25 +6,62 @@
...
+}
+
+@font-face {
+  font-family: "Roboto Mono";
+  src: url("/static/fonts/RobotoMono-Regular.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Artegra Sans Extended";
+  src: url("/static/fonts/ArtegraSansExtended-SemiBold.otf") format("opentype");
```
Comment (lukaw3d) [inline src/css/custom.css]: I doubt the license allows us to distribute this in a public repository

**Review by lukaw3d — CHANGES_REQUESTED** [review]

File: static/img/logo.svg (hunk 1 above)
Comment (matevz) [reply static/img/logo.svg]: Yeah, this is what all the cool kids in town now do. Usually it works. :stuck_out_tongue_closed_eyes:

File: src/css/custom.css [hunk 3]
```diff
@@ -6,25 +6,62 @@
...
+
+article strong {
+  font-family: "Roboto Bold", sans-serif;
+}
+
+article code {
+  font-family: "Roboto Mono", monospace;
+}
+
+h1, h2, h3, h4, h5 {
```
Comment (pro-wh) [inline src/css/custom.css]: can we do h1 only on this font

Comment (matevz) [reply src/css/custom.css]: Yeah, we can do that.

File: src/css/custom.css (hunk 2 above)
Comment (matevz) [reply src/css/custom.css]: I did some research. The Artegra studio is the author of the font:
https://artegrastudio.com/index.php/portfolio/artegra-superfamily/

And MyFonts is the official distributor:
https://www.myfonts.com/products/extend-semibold-artegra-sans-234675

The MyFonts licensing for webfonts is here:
https://www.myfonts.com/pages/faq#webfont-licensing

And is explicitly says:
> You will be serving the webfont kit for your own site and linking it in the CSS.

So I think hosting by us is fine as long as the license is there for our docs.oasis.io. I'll reconsider maybe using a different font though.

Comment (lukaw3d) [reply src/css/custom.css]: Hosting it on docs.oasis.io is fine, but not github

Comment (matevz) [reply src/css/custom.css]: Nvm, let's ditch the commercial font for now and just go with Roboto (Black) for headings.

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**